ansible/roles/ooni-collector/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+# FIXME: do we _really_ need nginx in front of the collector?
+- template: src=ngx-ooni-backend-web dest=/etc/nginx/sites-enabled/ooni-backend-web
+  notify: reload nginx
+
+# FIXME what is this? Do we need this for the collector?!
```

Comment:
**Question 2** What is this? Is this the configuration of the container network? Is this something we need for deploying the ooni collector role or is that something we can skip in this context?